defaultPnorLayoutSingleSide.xml

```diff
@@ -231,9 +231,17 @@ Layout Description
         <reprovision/>
     </section>
     <section>
+        <description>BMC_INV (36K)</description>
+        <eyeCatch>BMC_INV</eyeCatch>
+        <physicalOffset>0x1E75000</physicalOffset>
```

Comment:
I kept it consistent with the other layout files (defaultPnorLayoutWithoutGoldenSide.xml and defaultPnorLayoutWithGoldenSide.xml).   If you want me to move things around I would prefer to move the FIRDATA section to butt up against BMC_INV instead.   Thoughts?